mechanical_markdown/__init__.py

```diff
@@ -7,6 +7,6 @@
 from mechanical_markdown.recipe import Recipe as MechanicalMarkdown
 from mechanical_markdown.parsers import MarkdownAnnotationError
 
-__version__ = '0.5.2'
+__version__ = '0.5.3'
```

Comment:
Let's version this as 0.6.0. This is major enough!

```suggestion
__version__ = '0.6.0'
```